pkgs/by-name/oc/ocenaudio/package.nix

```diff
@@ -42,6 +42,12 @@ stdenv.mkDerivation rec {
     dpkg -x $src $out
     cp -av $out/opt/ocenaudio/* $out
     rm -rf $out/opt
+    mv $out/usr/share $out/share
+    rm -rf $out/usr
+    substituteInPlace $out/share/applications/ocenaudio.desktop \
+      --replace "/opt/ocenaudio/bin/ocenaudio" "ocenaudio"
+    mkdir -p $out/share/licenses/ocenaudio
+    mv $out/bin/ocenaudio_license.txt $out/share/licenses/ocenaudio/LICENSE
```

Comment:
That's a weird place to have the license originate from, but it is indeed correct.